app/controllers/search_controller.rb

```diff
@@ -119,21 +119,46 @@ def usernames
     render json: { result: [] }
   end
 
+  # TODO: [@rhymes] the homepage feed uses `feed_content_search` as an index,
+  # we should eventually move it to a JSON result
+  # in ArticlesController#Homepage or HomepageController#show
   def feed_content
-    feed_docs = if params[:class_name].blank?
-                  # If we are in the main feed and not filtering by type return
-                  # all articles, podcast episodes, and users
-                  feed_content_search.concat(user_search)
-                elsif params[:class_name] == "User"
-                  # No need to check for articles or podcast episodes if we know we only want users
-                  user_search
-                else
-                  # if params[:class_name] == PodcastEpisode, Article, or Comment then skip user lookup
-                  feed_content_search
-                end
+    class_name = params[:class_name].to_s.inquiry
```

Comment:
Ladies and gentlemen, the 2021 award for abusing `.inquiry` goes to *drumroll* @rhymes 🏆 😉 

```suggestion
    klass = params[:class_name].to_s.underscore.inquiry
```

Then you can check for `klass.article?` and `klass.user?` which look a lot less confusing.

Reply:
Hmm, the award may actually go to @atsmith813 after all... 😉

Reply:
Ah ah. I'm going to sync with @atsmith813 on this so we implement the same solution here and in https://github.com/forem/forem/pull/13401

Reply:
@citizen428 @jgaskins I see what you mean by it looking weird. I do find it _less_ confusing _without_ `underscore` because then we're matching the actual value instead of messing with an underscored version of `class_name` on the backend and a titleized version on the frontend.

Reply:
Let's keep it as it is, we can always change it later. We plan to remove `class_name` eventually and move each case in a different controller-action combo. After we removed Elasticsearch we'll have full control of the frontend and we can refactor both ends

Reply:
@jgaskins I think you may have tagged the wrong person.

@atsmith813 This may mean that `inquiry` just isn't the right choice here 😃 

@rhymes Agreed, we can leave it as-is for now.

Reply:
lol You're right, I did. 🤦🏼‍♂️ Friday was a weird day. 😂